### DISCUSSION TITLE
Subtitle Edit Recommendation/Request

### DISCUSSION THREAD

**Discussion by GitHubNoobN** [SubtitleEdit/subtitleedit#6846] [Ideas]
Hello,

Totally noob here. I hope I'm writing in the right place.

I'm trying to use the SE's ORC feature. When I select Batch Convert, I can only use the Binary Image Compare feature(i guess). However, tesseract gives much more accurate results. Is there a way to select tesseract when I'm using Batch Convert. (i guess not)

I noticed that the last line of most subtitles says which language they are subtitled in. I wish there was a way for SE to convert the image file of the last line to text, find out what language it is, and then make the necessary settings(select language, select tesseract settings, etc.) to allow us to use tesseract with Batch Convert feature.  :D

Ofcourse, it may be too difficult to do, unnecessary, or, due to technical obstacles, unfeasible. (if it has not already been done) But, like I said, I'm noob. And, when it comes to coding, I have to study for a few years just to be a noob (seriously).

However, I think I know the mindset of coders. In their world, it's a request for sure. Nevertheless, from my point of view, this is a very useful suggestion(if it's not a recommendation). And, so, I wanted to share it with the community. Hopefully it will help.

Kind Regards

**Comment by Purfview** [comment]
Dunno what your issue is, but let's clarify some things:
If you are OK with <ins>inaccurate</ins> OCR then use Tesseract, use  the Binary Image Compare or nOCR if you want <ins>accurate</ins> OCR.

**Reply by GitHubNoobN** [reply]
Thanks for reply, Purfview. I might be doing something wrong for Binary Image Compare to work right; or I might be doing something accidentally right for Tesseract to work relatively well. I will check to the best of my knowledge for sure. What do you think I could be doing wrong?(I'm using Subtitle Edit 3.6.12.0 and Windows 8.1, if that has anything to do with it.)

**Reply by Purfview** [reply]
Binary Image Compare is for smaller DVD like subtitles, for bigger like BluRay subs use nOCR.
You train these methods on a go, it's hard to tell what you are doing wrong when you didn't described what and how you do.

**Comment by la7bab** [comment]
plz add HC-ESub hardcoded subtitles extraction

**Reply by Purfview** [reply]
> plz add HC-ESub hardcoded subtitles extraction

That's way out of this program's scope.
You can extract them with this: https://github.com/Purfview/InpaintDelogo
